package.json

```diff
@@ -7,7 +7,8 @@
     "husky": "^3.0.0",
     "lerna": "^3.10.8",
     "truffle-privatekey-provider": "1.1.0",
-    "wait-on": "^3.2.0"
+    "wait-on": "^3.2.0",
+    "web3": "^1.2.4"
```

Comment:
Maybe we should consider moving the cypress suite to a sub-package to avoid polluting the mono-repo root dependencies.

Reply:
After further thought I'm not sure this makes sense. I think if this causes any issues we can reevaluate.